README.md

```diff
@@ -1,2 +1,65 @@
-# mint.com-signup-clone
-This project consists of building an HTML document that matches the appearance of mint.com’s signup page
+# Mint.com Signup Clone
+> This project consists of building an HTML document that matches the appearance of mint.com’s signup page
+
```

Comment:
- Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

Reply:
Thanks for your valuable feedback. We will improve our Readme file as per your suggestions.